sample/src/main/java/com/example/api/MerchantApi.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.api
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+
+/**
+ * The use of Retrofit + Moshi here is entirely a convenience. Use the API stack you are
+ * most comfortable with.
+ */
+interface MerchantApi {
```

Comment:
This isn't used in the sample yet (It'll come into play for adding V1 / V2 to sample app)